docs/EN_US/ContainerizedHPCC/ContainerizedMods/LocalDeployment.xml

```diff
@@ -163,31 +185,287 @@ sasha-wu-archiver     ClusterIP    10.111.34.240   &lt;none&gt;       8877/TCP
+          <varlistentry>
+            <term>Note:</term>
+
+            <listitem>
+              <para>The value passed to --set common-hostpath is case
+              sensitive.</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+      </listitem>
+
+      <listitem>
+        <para>Copy the output from the helm install command in the previous
+        step from the word <emphasis role="bold">storage:</emphasis> to the
+        end, and save it to a text file.</para>
+
+        <para>In this example, we will call the file mystorage.yaml. The file
+        should look similar to this:</para>
+
+        <para><programlisting>storage:
+  planes:
+  - name: dali
+    pvc: dali-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/dalistorage"
+    category: dali
+  - name: dll
+    pvc: dll-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/queries"
+    category: dll
+  - name: sasha
+    pvc: sasha-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/sasha"
+    category: sasha
+  - name: debug
+    pvc: debug-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/debug"
+    category: debug
+  - name: data
+    pvc: data-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/hpcc-data"
+    category: data
+  - name: mydropzone
+    pvc: mydropzone-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/dropzone"
+    category: lz
+
+sasha:
+  wu-archiver:
+    plane: sasha
+  dfuwu-archiver:
+    plane: sasha</programlisting></para>
+      </listitem>
+
+      <listitem>
+        <para>If you are using Docker Desktop with Hyper-V, add the shared
+        data folder (in this example, C:\hpccdata) in Docker Desktop's
+        settings by pressing the Add button and typing c:\hpccdata.</para>
+
+        <para>This is <emphasis role="bold">not</emphasis> needed in a MacOS
+        or WSL 2 environment.</para>
+
+        <graphic fileref="../../images/dockerFileShare.jpg" />
+      </listitem>
+
+      <listitem>
+        <para>Finally, install the hpcc Helm chart, and provide a yaml file
+        that provides the storage information created by the previous
+        step.</para>
+
+        <programlisting>helm install mycluster hpcc/hpcc --version-f mystorage.yaml --version=8.6.14</programlisting>
+
+        <variablelist>
+          <varlistentry>
+            <term>Note:</term>
+
+            <listitem>
+              <para>The --version argument is optional, but recommended. It
+              ensures that you know which version you are installing. If
+              omitted, the latest non-development version is installed. This
+              example uses 8.614, but you should use the version you want.
+              </para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+      </listitem>
+
+      <listitem>
+        <para>To test, access the ECL Watch Playground and create some data
+        files and workunits by submitting to Thor some ECL code like the
+        following:</para>
```

Comment:
minor: could this sentence be improved breaking out into couple steps. (as you did on line 191 above)